cmd/beacon-chain/flags/config.go

```diff
@@ -78,3 +96,21 @@ func configureMinimumPeers(ctx *cli.Context, cfg *GlobalFlags) {
 		cfg.MinimumSyncPeers = maxPeers
 	}
 }
+
+func validateStateDiffExponents(exponents []int) error {
```

Comment:
Could you add a godoc here that explains these constraints?

Why does it have to be between 1 and 15 values? Why does the last exponent have to be greater than 5.

Isn't there some overflow check you should do here too? 2^max_int seems like an invalid input

Reply:
The feedback was to answer those questions in the godoc